osbuild.spec

```diff
@@ -325,6 +329,10 @@ fi
 %{pkgdir}/solver.json
 
 %changelog
+* Thu Apr 17 2025 Benjamin A. Beasley <code@musicinmybrain.net> - 148-2
+- In Fedora 41+, depend on python3-typer-slim rather than python3-typer, working
+  around a file conflict
+
```

Comment:
Ideally, I'd suggest omitting the changelog entry as well. Your contribution will be attributed to you in the git history. The changelog from the upstream SPEC is not used anywhere, except for development builds done in COPR. Fedora SPEC file's changelog is always.

Reply:
Makes sense. I’ll adjust this and force-push.